node/src/BaseClient.ts

```diff
@@ -1210,7 +1239,8 @@ export class BaseClient {
         );
     }
 
-    /** Retrieve the values of multiple keys.
+    /**
+     * Retrieves the values of multiple keys.
```

Comment:
would be really good to move all these peripheral changes to another PR
so we can focus on what's important:  the Record refactor

Reply:
Reverted